carp.common/src/commonTest/kotlin/dk/cachet/carp/common/application/TestInstances.kt

```diff
@@ -49,7 +49,7 @@ val commonInstances = listOf(
     Smartphone( "User's phone" ),
 
     // Shared device registrations in `devices` namespace.
-    DefaultDeviceRegistration( "Some device" ),
+    DefaultDeviceRegistration(),
```

Comment:
At first sight, it's not clear to me why you remove parameters when constructing `DefaultDeviceRegistration` everywhere in tests

Reply:
Long story.

I was playing around/considering whether `deviceId` and `deviceDisplayName` should be optional or not, and that also impacted the "logical" ordering of them as parameters of `DefaultDeviceRegistration`.

From an API perspective, I concluded it makes more sense to have `deviceDisplayName` as the first parameter, and that in the majority of use cases you generate a `deviceId`.

This resulted in me having to update the tests several times, which I grew tired off. :) So, I settled on these parameters being optional, and non-essential for tests, which I committed in the first commit, to only then include `deviceDisplayName` without having to modify the tests.